source/core/utils/get-buffer.ts

```diff
@@ -0,0 +1,15 @@
+import {Readable} from 'stream';
+
+// TODO: Update `get-stream`
+
+const getBuffer = async (stream: Readable) => {
+	const chunks = [];
+
+	for await (const chunk of stream) {
+		chunks.push(Buffer.from(chunk));
+	}
+
+	return Buffer.concat(chunks);
```

Comment:
I would recommend providing the total length as it can slightly improve performance. https://github.com/sindresorhus/get-stdin/blob/master/index.js